packages/terra-heading/src/terra-dev-site/doc/example/colors.module.scss

```diff
@@ -0,0 +1,24 @@
+:root {
+  --terra-heading-attention-color: rgb(220, 20, 60);
+  --terra-heading-neutral-color: rgb(46, 125, 50);
+  --terra-heading-info-color: #304ffe;
+  --terra-heading-default-color: #333;
+}
+
+:local {
+  .default {
+    color: var(--terra-heading-default-color, #333);
+  }
+
+  .attention {
+    color: var(--terra-heading-attention-color, rgb(220, 20, 60));
+  }
+
+  .neutral {
```

Comment:
```suggestion
  .success {
```

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/618ae6178edc4a734ef31a044c3bc8d570051b5c